internal/ledger/compare_test.go

```diff
@@ -498,13 +484,11 @@ func TestJSONArrayFileWriter(t *testing.T) {
 				"value" : "green",
 				"blockNum" : 472,
 				"txNum" : 61
-			},
-			"snapshotrecord2" : {}
```

Comment:
This was an intentional entry that was chosen to represent a missing record. Unless there is a strong reason, I would prefer to keep it as is.